packages/svelte-ux/src/lib/components/LanguageSelect.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import Button from './Button.svelte';
+  import Menu from './Menu.svelte';
+  import MenuItem from './MenuItem.svelte';
+  import { cls } from '../utils/styles';
+  import { getSettings } from './settings';
+  const { locale } = getSettings();
+
+  let open = false;
+
+  type Language = {
+    name: string;
+    code: string;
+    flag: string;
+  };
+
+  export let languagesDemo: Language[] = [
+    { name: 'English', code: 'en', flag: '🇺🇸' },
+    { name: 'Français', code: 'fr', flag: '🇫🇷' },
+    // add more for the demo
+  ];
+  $: languageSelected = languagesDemo.find((c) => c.code === $locale)!;
+</script>
+
+<Button on:click={() => (open = !open)}>
+  {languageSelected.flag}
+  <Menu bind:open on:close={() => (open = false)} offset={4} explicitClose resize>
+    <div class="grid gap-2 p-2 border-b border-surface-content/10">
+      {#each languagesDemo as language}
+        <MenuItem
+          on:click={() => {
+            languageSelected = language;
+            locale.set(language.code);
```

Comment:
This is changing the `locale`, but when I'm on the DatePicker I don't see any change.

Reply:
hmm.. I noticed this as well (same for the other Date* components)..  Have you looked into it yet?